.codesandbox/ci.json

```diff
@@ -0,0 +1,3 @@
+{
+  "sandboxes": ["/.codesandbox/sandbox"]
```

Comment:
If the examples in the browser package still exist, possibly you could just reuse those. But a separate demonstration works as well of course.

Reply:
The examples still exist, but I actually wanted to replace that code with the one featured in this PR eventually: the React hooks make the code much cleaner, both for `useEffect` and `useState`. 

Also for some reason that I didn't take the time to investigate yet, the parcel bundler complains about missing dependencies that don't prevent webpack from building the examples, so I'm not sure the exact same code would run in both contexts.